File: Programmers/suyeon/구명보트.py
def solution(people, limit):
    people.sort()
    
    answer = 0
    left, right = 0, len(people) - 1
    
    while left <= right:
        answer += 1
        
        if people[left] + people[right] <= limit:
            left += 1
        
        right -= 1
        
    return answer

File: Programmers/hyeonyoung/구명보트.cpp
#include <string>
#include <vector>
#include <algorithm>

using namespace std;

int solution(vector<int> people, int limit)
{
    int answer = 0;

    sort(people.begin(), people.end());
    int x = 0, y = people.size() - 1;
    while (x <= y)
    {
        if (people[x] + people[y] <= limit)
        {
            x++;
            y--;
            answer++;
        }
        else
        {
            y--;
            answer++;
        }
    }

    return answer;
}
